docs/src/main/paradox/r2dbc.md

```diff
@@ -47,14 +47,20 @@ The table below shows `akka-projection-r2dbc`'s direct dependencies, and the sec
 ## Schema
 
 The `akka_projection_offset_store`, `akka_projection_timestamp_offset_store` and `akka_projection_management` tables
-need to be created in the configured database, unless using H2 where the schema is created automatically:
+need to be created in the configured database:
 
 PostgreSQL
 :  @@snip [PostgreSQL Schema](/akka-projection-r2dbc/ddl-scripts/create_tables_postgres.sql)
 
 YugaByte
 :  @@snip [YugaByte Schema](/akka-projection-r2dbc/ddl-scripts/create_tables_yugabyte.sql)
 
+For H2 the schema need to be defined as the `additional-init` setting in your config. This means it is created on first
+connection instead of up front (needed as there is no way to connect to the database from outside the JVM process):
+
+H2
+:  @@snip [H2 Schema](/akka-projection-r2dbc/src/it/resources/application-h2.conf) { #schema }
```

Comment:
I think this looks better as a H2 tab alongside the others even though it's config:

```
PostgreSQL
:  @@snip [PostgreSQL Schema](/akka-projection-r2dbc/ddl-scripts/create_tables_postgres.sql)

YugaByte
:  @@snip [YugaByte Schema](/akka-projection-r2dbc/ddl-scripts/create_tables_yugabyte.sql)

H2
:  @@snip [H2 Schema](/akka-projection-r2dbc/src/it/resources/application-h2.conf) { #schema }
```

For H2 the schema need to be defined as the `additional-init` setting in your config. This means it is created on first connection instead of up front (needed as there is no way to connect to the database from outside the JVM process).

Reply:
Alright, was on the fence with that and that it is confusing that it's not pure DDL, but you are probably right.

Reply:
fixed in [5c03be1](https://github.com/akka/akka-projection/pull/914/commits/5c03be12e68c26a29a67b4f6f10c9d33277403f6)